src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/avoidescapedunicodecharacters/InputAvoidEscapedUnicodeCharactersEscapedS.java

```diff
@@ -0,0 +1,28 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.avoidescapedunicodecharacters;
+
+/* Config:
+ *
+ * allowEscapesForControlCharacters = false
+ * allowByTailComment = false
+ * allowIfAllCharactersEscaped = true
+ * allowNonPrintableEscapes = false
+ */
+public class InputAvoidEscapedUnicodeCharactersEscapedS {
+    String value1 = "\u03bc\t"; // ok
+    String value2 = "\u03bc\s"; // ok
+    String value3 = "\u03bc\s not all escaped chars"; // violation
+    String value31 = "\u03bc\n not all escaped chars"; // violation
+    String value4 = """
+            \s\s\s\n
+            """; // ok
+    String value5 = """
+            \u03bc\s
+            """; // ok
+    String value6 = """
+            \s\s\s\n not all escaped chars
+            """; // violation
+    String value7 = """
+            \u03bc\s not all escaped chars
+            """; // violation
```

Comment:
`// violation` comments should be on lines 19 and 25, where violation is reported

Reply:
@strkkk it is not possible to add comments to those particular lines; they would become text block content. I have added comments to make that more clear in the test case.

Reply:
It is ok to to have `// violation` as part of string value. All we need is visual marker that line has violation

Reply:
@romani in the case of text blocks, the initial newline and triple quotes delimit the content. I cannot put a comment or text following the delimiter on the same line.  I could do a C style comment:
```
                String test = /* violation */"""
                        """;
```

Reply:
I left a comment after each text block, and a C style comment on the same line.